Makefile

```diff
@@ -58,7 +58,7 @@ setup:
 
 .PHONY: test
 test:
-	$(ISTANBUL) cover node_modules/.bin/_mocha -- --recursive
+	$(ISTANBUL) cover node_modules/.bin/_mocha -- --recursive --timeout 10000
```

Comment:
Interesting, I've seen these failures locally and just thought it was to do with the terrible laptop I'm using.